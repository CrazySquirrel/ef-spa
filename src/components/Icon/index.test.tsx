import * as React from 'react';
import {shallow} from 'enzyme';

import {IconChat} from './index';

describe('IconChat', () => {
  it('render', () => {
    const result = shallow(<IconChat/>);

    expect(result).toMatchSnapshot();
  });
});
