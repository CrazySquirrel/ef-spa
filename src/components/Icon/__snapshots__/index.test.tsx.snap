version https://git-lfs.github.com/spec/v1
oid sha256:ad8ce5a818b7ec0817efc0fdb6141ee700f7edd02f73143d7c1fc8bbbfcdb505
size 2032
